AtApi Solution/AtApi/Models/AccountViewModels/RegisterViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AtApi.Models.AccountViewModels
+{
+    public class RegisterViewModel
+    {
+        [Required]
+        [EmailAddress]
+        [Display(Name = "Email")]
+        public string Email { get; set; }
+
+        [Required]
+        [StringLength(100, ErrorMessage = "The {0} must be at least {2} and at max {1} characters long.", MinimumLength = 6)]
+        [DataType(DataType.Password)]
+        [Display(Name = "Password")]
+        public string Password { get; set; }
+
+        [DataType(DataType.Password)]
+        [Display(Name = "Confirm password")]
+        [Compare("Password", ErrorMessage = "The password and confirmation password do not match.")]
+        public string ConfirmPassword { get; set; }
+    }
```

Comment:
We will also have "agreed to terms and conditions", first name and last name of the user upon registration.

Reply:
Would that be more of a client confirmation? They wouldn't be registered unless they've agreed to the T&C.

Reply:
It should be on both - client side and server side.

Reply:
Am I understanding this correctly - RegisterViewModel is the model for when a user is registering for the first time? They must agree to terms and conditions. Client will ensure of that but BE should also verify in case a user is a hacker and bypasses UI completely and sends the request directly to BE